### PR TITLE
Boost Control Draft

### PR DESCRIPTION
**Pull request by 960**
The TS Part only

### CODE REVIEW

**Comment by 960** [comment]
The difference.

AiEkVxuRdH

**Comment by 960** [comment]
The difference made:

javaw_FI6mUdSdI7

File: firmware/tunerstudio/rusefi.input [hunk 1]
```diff
@@ -2498,14 +2507,14 @@ cmd_set_engine_type_default					= "w\x00\x31\x00\x00"
 		field = "Two-wire mode",						etb_use_two_wires, {throttlePedalPositionAdcChannel != 16}
 		field = "PWM Frequency",						etbFreq, {throttlePedalPositionAdcChannel != 16}
 		field = etbCalibrationOnStart,					etbCalibrationOnStart
-		field = "No1 Direction #1",						etb1_directionPin1, {throttlePedalPositionAdcChannel != 16}
-		field = "No1 Direction #2",						etb1_directionPin2, {throttlePedalPositionAdcChannel != 16}
-		field = "No1 Control #1",						etb1_controlPin1, {throttlePedalPositionAdcChannel != 16 && etb_use_two_wires == 0}
-		field = "No2 Direction #1",						etb2_directionPin1, {throttlePedalPositionAdcChannel != 16}
-		field = "No2 Direction #2",						etb2_directionPin2, {throttlePedalPositionAdcChannel != 16}
+		field = "No1 Direction #1",						etbIo1_directionPin1, {throttlePedalPositionAdcChannel != 16}
```
Comment (rusefi) [inline firmware/tunerstudio/rusefi.input]: you are making a huge mistake here. Your "boost control" draft is messing with ETB settings.

Comment (960) [reply firmware/tunerstudio/rusefi.input]: I can change them back, but the old pin-names doesnt work anymore..

File: firmware/tunerstudio/rusefi.input [hunk 2]
```diff
@@ -2191,7 +2198,9 @@ cmd_set_engine_type_default					= "w\x00\x31\x00\x00"
 ;			Board->Connection
 	dialog = tsPort, "TunerStudio Port"
 		field = "tunerStudioSerialSpeed",				tunerStudioSerialSpeed
-		field = "Use PC10/PC11 serial?",				useSerialPort
+		field = "Use UART/TTL serial?",					useSerialPort
```
Comment (mck1117) [inline firmware/tunerstudio/rusefi.input]: unrelated change?

Comment (960) [reply firmware/tunerstudio/rusefi.input]: I pulled this file again to remove some bug fixes, I tought I pasted only the boost control.
Are this a diff now?

Comment (960) [reply firmware/tunerstudio/rusefi.input]: It had been edited in the meantime, from the one I originally pushed.

I dont know git good enough to know if this is a current difference..
